configs/stage/schemas/src/compliance.ksl

```diff
@@ -0,0 +1,29 @@
+version 0.1
+namespace compliance
+
+import rbac
+
+// Policy
+@rbac.add_v1_based_permission(app:'compliance', resource:'policy', verb:'read', v2_perm:'compliance_policy_view');
+@rbac.add_v1_based_permission(app:'compliance', resource:'policy', verb:'create', v2_perm:'compliance_policy_new');
+@rbac.add_v1_based_permission(app:'compliance', resource:'policy', verb:'write', v2_perm:'compliance_policy_edit');
+@rbac.add_v1_based_permission(app:'compliance', resource:'policy', verb:'delete', v2_perm:'compliance_policy_remove');
+
+// This permission is not used by compliance backend
+// But frontend uses it here: https://github.com/RedHatInsights/compliance-frontend/blob/cd020f48920f51f0422256e7d91beada13e5d5d4/src/Routes.js#L103
+// And it is part of the roles: https://github.com/search?q=org%3ARedHatInsights+compliance%3Apolicy%3Aupdate&type=code
+@rbac.add_v1only_permission(perm:'compliance_policy_update');
+
+// https://github.com/RedHatInsights/compliance-frontend/blob/cd020f48920f51f0422256e7d91beada13e5d5d4/src/Routes.js#L38
+// https://github.com/search?q=org%3ARedHatInsights%20compliance%3Areport%3Adelete&type=code
+@rbac.add_v1only_permission(perm:'compliance_report_delete');
```

Comment:
I think this is fine for now, but note that v1only permissions are essentially just containers - the permissions here won't be accessible via Kessel until they're mapped to something V2.

Reply:
These are not used anymore